ide/coqide/dune

```diff
@@ -31,7 +31,7 @@
   (language c)
   (names coqide_os_stubs))
  (optional)
- (libraries coqide-server.protocol coqide-server.core lablgtk3-sourceview3))
+ (libraries coq-core.toplevel coqide-server.protocol coqide-server.core lablgtk3-sourceview3))
```

Comment:
Ups I'm afraid that this is going too far :( as it will link the whole of Coq to coqide. There are several ways to avoid this, we can discuss on Zulip if you want.

Reply:
And actually it is linking coqide-server.core already, that seems wrong too! Let me see the history. I hope I'm not just confused.